.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,44 @@
   - when this PR is merged.
   -->
```

Comment:
Let's maybe have these first few lines follow exactly lines 1-9 of https://github.com/oppia/oppia/edit/develop/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Done.